Framework/PythonInterface/mantid/plots/axesfunctions.py

```diff
@@ -433,8 +433,12 @@ def _pcolorpieces(axes, workspace, distribution, *args, **kwargs):
     maxi = numpy.max([numpy.max(i) for i in z])
     if 'vmin' in kwargs:
         mini = kwargs['vmin']
+        # Passing normalized and vmin or vmax is not supported in matplotlib
+        del kwargs['vmin']
```

Comment:
Should we warn or something?
I'm worried about us silently discarding args and the fact it can be confusing for users

Reply:
These were previously supported args in matplotlib and no longer are, however, our API still supports and lightly depends on it. So to users, there should be little to no changes even in scripts.

Reply:
I agree, hence a warning not an error to hint to users that this won't do what they expect it to anymore?

Reply:
It still does what it used to inside our code, i.e. setting mini and maxi variables to this value if given, it's just removed from kwargs before being passed to matplotlib, which was an invalid input originally anyway. I am not sure what a good warning would look like, "Can't pass vmin or vmax to matplotlib pcolormesh alongside normalisation, but mantid still behaves the same", do you have a better idea?

Reply:
Oh if we already warn internally elsewhere then that's fine :+1: